### ISSUE TITLE
Editor axis adjusters are glitching

### ISSUE THREAD

**Issue by GamingBud** [defold/editor2-issues#2080]
I’ve had this issue for a while, but I figured it would be reported by someone else and patched in the next update. It’s been hanging on for the past few updates, so I’m pretty sure I’m the only one who has this issue.

It’s the same story as my most recent issue, the (null)(null) error. I don’t know what I did, or what happened.

![2ecx0l](https://user-images.githubusercontent.com/11013079/42982824-d457c446-8ba8-11e8-91b2-0278a20696dd.gif)

As you can see, the axis markers and multi-select box aren’t working correctly. They both appear to be running into contact points all over the grid screen.

<hr/>

<table>
<tr><td>Build time</td><td>2018-07-18T17:37:42.353005</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>4828309a21aaec8b7e6d80351d863c6a15a486d9</td></tr>
<tr><td>Defold engine sha</td><td>7b2c2c019d6fa106f78e2e98cd3009a21d4095aa</td></tr>
<tr><td>Defold version</td><td>1.2.133</td></tr>
<tr><td>GPU</td><td>Radeon(TM) RX 460 Graphics</td></tr>
<tr><td>GPU Driver</td><td>4.5.14523 Compatibility Profile Context 24.20.12019.1010</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 7</td></tr>
<tr><td>OS version</td><td>6.1</td></tr>
<table>

**Comment by GamingBud** [comment]
Any idea what the issue is?

**Comment by subsoap** [comment]
Does the thing you are editing have a custom material?

**Comment by GamingBud** [comment]
@subsoap Nope, all defaults. I do have Rendercam in the project, but that shouldn't mess with the editor.

**Comment by subsoap** [comment]
I had this happen and it was likely due to a buggy .dae file. What is the file you are trying to move?

**Comment by subsoap** [comment]
Hmm, this also happens with apparently non-broken .dae file. Here is sample project, try to move the GOs with a model attached

[3DModelTest.zip](https://github.com/defold/editor2-issues/files/2238402/3DModelTest.zip)

**Comment by GamingBud** [comment]
I can't even move the objects. It crashes a few seconds after selecting an object as a menu item. Tried 4 times.
